core/src/test/java/io/questdb/cutlass/line/tcp/LineTcpCommitFuzzTest.java

```diff
@@ -86,7 +86,7 @@ public void testCommitIntervalBasedDefaultLagZero() throws Exception {
     @Test
     public void testCommitIntervalBasedFraction() throws Exception {
         // This test makes sense for only non-WAL tables since they support commit lag.
-//        Assume.assumeFalse(walEnabled);
+        Assume.assumeFalse(walEnabled);
```

Comment:
why not run for WAL?

Reply:
all tests were enabled in the latest commit

Reply:
this is the opposite of enabling a test, this is explicitly `disabling the test` for WAL

Reply:
you are looking at old code :)